### ISSUE TITLE
Unable to archive contentlet with workflow API in a starter instance

### ISSUE THREAD

**Issue by muhammadfaizandotcms** [dotCMS/core#23754]
### Proposed Objective

Accessibility (WCAG)

### Proposed Priority

Priority 1 - Show Stopper

### dotCMS Version

22.03.02

### Problem Statement

The user is Unable to archive contentlet with workflow API in a starter instance.
A backend role or CMS User role is needed to archive the contentlet with a starter instance.
However, with the same permissions in a blank instance user is able to archive the contentlet.
Relevant Ticket: https://dotcms.zendesk.com/agent/tickets/109497
Here is the CURL command.
```
curl --location --request PUT 'https://demo.dotcms.com/api/v1/workflow/actions/fire?identifier=c0d2986751a19f57d44061ec08ec1241' \
--header 'Authorization: Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJhcGk1MzQ4YjkxNy1hOWE1LTQ0NGYtOWVhYS0zMDI2NTc4NTY4NjkiLCJ4bW9kIjoxNjczMDE2Njc0MDAwLCJuYmYiOjE2NzMwMTY2NzQsImlzcyI6IjUzZjJlYWRkODAiLCJleHAiOjE3Njc2Mzk2MDAsImlhdCI6MTY3MzAxNjY3NCwianRpIjoiMjU3NmRmZTYtMzMzYS00MjAwLTlhMjktZDIyNGMyYzA5NDVlIn0.K5S7PhKajwB3PyrBmrugi4u0pD4yqpcwbSKSoUunZT0' \
--header 'Content-Type: application/json' \
--header 'Cookie: dmid=b903cb54-30aa-471f-90c2-039abc134036' \
--data-raw '{
    "actionName":"Archive"
}'
```

### Steps to Reproduce

1. Go to the demo site.
2. Create a test user, test role, and test content type.
3. Add a test role to the workflow subaction who can use the list.
4. Assign permissions to test the role according to the screenshots.
5. Call the workflow API.
6. see the error. User 'user-0a203e85-3d9a-4822-bdd7-0a5e146cab42' does not have READ permissions on Contentlet [name: One, type: Test, lang: 1, identifier: c0d2986751a19f57d44061ec08ec1241, inode: 2ba02f61-fb14-4724-a48a-2e753f42a0f4]workflow

### Acceptance Criteria

The user should be able to archive the contentlet with the starter instance just like the blank instance.

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

![Screenshot from 2023-01-06 22-16-18](https://user-images.githubusercontent.com/103441252/211063692-c6e2acaf-807c-42f1-b317-78278e2f35d2.png)
![Screenshot from 2023-01-06 22-16-33](https://user-images.githubusercontent.com/103441252/211063696-fe7f5f50-477c-4ba5-a8cc-22fd9b5d714c.png)
![Screenshot from 2023-01-06 22-16-42](https://user-images.githubusercontent.com/103441252/211063699-09e47906-674c-4231-bb64-d2e617428e74.png)
![Screenshot from 2023-01-06 22-16-59](https://user-images.githubusercontent.com/103441252/211063701-0a8379a7-2ea9-45ec-afeb-5faaa701100c.png)

**Comment by damen-dotcms** [comment]
Backend users are required to interact with unpublished content.

**Comment by wezell** [comment]
A backend user is required to manage content in dotCMS.  This is by design. Front end users are not permitted to read/manage non-live content.
